2021-05-27 23:28:32,181 INFO    MainThread:22323 [wandb_setup.py:_flush():69] setting env: {}
2021-05-27 23:28:32,181 INFO    MainThread:22323 [wandb_setup.py:_flush():69] setting login settings: {}
2021-05-27 23:28:32,182 INFO    MainThread:22323 [wandb_init.py:_log_setup():336] Logging user logs to /opt/ml/gitclone/p4-mod-model_diet/wandb/run-20210527_232832-394qptn8/logs/debug.log
2021-05-27 23:28:32,182 INFO    MainThread:22323 [wandb_init.py:_log_setup():337] Logging internal logs to /opt/ml/gitclone/p4-mod-model_diet/wandb/run-20210527_232832-394qptn8/logs/debug-internal.log
2021-05-27 23:28:32,182 INFO    MainThread:22323 [wandb_init.py:init():369] calling init triggers
2021-05-27 23:28:32,182 INFO    MainThread:22323 [wandb_init.py:init():374] wandb.init called with sweep_config: {}
config: {'img_size': 2, 'batch_size': 8, 'init_lr': 0.0006703173457965114}
2021-05-27 23:28:32,182 INFO    MainThread:22323 [wandb_init.py:init():418] starting backend
2021-05-27 23:28:32,182 INFO    MainThread:22323 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-05-27 23:28:32,192 INFO    MainThread:22323 [backend.py:ensure_launched():132] starting backend process...
2021-05-27 23:28:32,199 INFO    MainThread:22323 [backend.py:ensure_launched():136] started backend process with pid: 22332
2021-05-27 23:28:32,200 INFO    MainThread:22323 [wandb_init.py:init():423] backend started and connected
2021-05-27 23:28:32,201 INFO    MainThread:22323 [wandb_init.py:init():465] updated telemetry
2021-05-27 23:28:32,202 INFO    MainThread:22323 [wandb_init.py:init():484] communicating current version
2021-05-27 23:28:32,956 INFO    MainThread:22323 [wandb_init.py:init():489] got version response upgrade_message: "wandb version 0.10.31 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-05-27 23:28:32,956 INFO    MainThread:22323 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-05-27 23:28:33,190 INFO    MainThread:22323 [wandb_init.py:init():522] starting run threads in backend
2021-05-27 23:28:33,225 INFO    MainThread:22323 [wandb_run.py:_console_start():1565] atexit reg
2021-05-27 23:28:33,226 INFO    MainThread:22323 [wandb_run.py:_redirect():1412] redirect: SettingsConsole.REDIRECT
2021-05-27 23:28:33,226 INFO    MainThread:22323 [wandb_run.py:_redirect():1417] Redirecting console.
2021-05-27 23:28:33,228 INFO    MainThread:22323 [wandb_run.py:_redirect():1479] Redirects installed.
2021-05-27 23:28:33,228 INFO    MainThread:22323 [wandb_init.py:init():546] run started, returning control to user process
2021-05-27 23:28:36,391 INFO    MainThread:22323 [wandb_run.py:_atexit_cleanup():1535] got exitcode: 1
2021-05-27 23:28:36,391 INFO    MainThread:22323 [wandb_run.py:_restore():1507] restore
2021-05-27 23:28:38,398 INFO    MainThread:22323 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 717
  total_bytes: 717
}

2021-05-27 23:28:38,649 INFO    MainThread:22323 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 717
  total_bytes: 717
}

2021-05-27 23:28:39,212 INFO    MainThread:22323 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 717
  total_bytes: 3833
}

2021-05-27 23:28:39,613 INFO    MainThread:22323 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3835
  total_bytes: 3835
}

2021-05-27 23:28:39,714 INFO    MainThread:22323 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3835
  total_bytes: 3835
}

2021-05-27 23:28:39,816 INFO    MainThread:22323 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3835
  total_bytes: 3835
}

2021-05-27 23:28:39,918 INFO    MainThread:22323 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3835
  total_bytes: 3835
}

2021-05-27 23:28:40,019 INFO    MainThread:22323 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3835
  total_bytes: 3835
}

2021-05-27 23:28:40,121 INFO    MainThread:22323 [wandb_run.py:_wait_for_finish():1657] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3835
  total_bytes: 3835
}

2021-05-27 23:28:41,196 INFO    MainThread:22323 [wandb_run.py:_show_files():1879] logging synced files
